src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/nonassociations/NonAssociationsReconciliationService.kt

```diff
@@ -29,25 +29,24 @@ class NonAssociationsReconciliationService(
   private val pageSize: Long,
 ) {
 
-  var nomisTotalDetails = 0
```

Comment:
private variable in the service that was keeping a count of totals across `async` threads